dataprofiler/profilers/column_profile_compilers.py

```diff
@@ -194,6 +209,62 @@ def profile(self):
             )
         return profile
 
+    def diff(self, other, options=None):
+        """
+        Finds the difference between 2 compilers and returns the report
+        
+        :param other: profile compiler finding the difference with this one.
+        :type other: ColumnPrimitiveTypeProfileCompiler
+        :return: difference of the profiles
+        :rtype: dict
+        """
+        # Call super for compiler instance check
+        super().diff(other)
+
+        # Initialize profile diff dict
+        profile = {
```

Comment:
thoughts on changing the name to `diff_profile`?

Reply:
done